backend/app/assets/stylesheets/spree/backend/shared/_typography.scss

```diff
@@ -98,6 +98,8 @@ dl {
 .red    { color: $color-5 }
 .yellow { color: $color-6 }
 
+.negative { color: $color-5 !important }
```

Comment:
Could we use a meaningful color variable (we have `$red`) here?

Reply:
Do we really need the `!important` here? Especially with this generic class this could cause headaches. Would a stronger selector like

```css
input[type="number"].negative
```

work as well?